vite.config1.js

```diff
@@ -0,0 +1,89 @@
+import { defineConfig } from 'vite';
```

Comment:
No description why we have to use this, is it possible to merge it with original config? Use english please.